LibTest/ffi/LongLong/LongLong_A02_t01.dart

```diff
@@ -13,17 +13,17 @@
 /// and [Union] fields.
 ///
 /// @description Checks that this type is a signed 64-bit integer
-/// @author sgrekhov@unipro.ru
+/// @author sgrekhov22@gmail.com
```

Comment:
I'm not sure if it makes sense to update an author. (Even if it's the same person.) @eernstg  ?

Reply:
The same as the above. Just forgot to change the author of the new test (which was produced by copy-paste of an existing file and then changing the content)

Reply:
I agree, this is the right thing to do.